admin/htop/Makefile

```diff
@@ -8,17 +8,19 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=htop
-PKG_VERSION:=2.2.0
-PKG_RELEASE:=3
+PKG_VERSION:=3.0.0
+PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://hisham.hm/htop/releases/$(PKG_VERSION)/
-PKG_HASH:=d9d6826f10ce3887950d709b53ee1d8c1849a70fa38e91d5896ad8cbc6ba3c57
+PKG_SOURCE_URL:=https://github.com/htop-dev/htop/archive/$(PKG_VERSION)/
```

Comment:
use codeload for the URL instead:

```
https://codeload.github.com/htop-dev/htop/tar.gz/$(PKG_VERSION)?
```

Reply:
I get 403 invalid when I use the codeland. code you suggested.

Reply:
Never tried codeland.... Is the literal ? at the end intended to be there or should it be a forward slash?

Reply:
Renames the file. That URL generates 3.0.0.tar.gz , which can conflict with other packages.

The typical way to fix this is to use ? and let the build system rename it.

Another way to fix this is to use curl with the -J parameter. I haven't seen anyone wanting to do this.